ownCloud/Settings/SecuritySettingsSection.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  SecurityGlobalSettings.swift
+//  ownCloud
+//
+//  Created by Pablo Carrascal on 30/04/2018.
+//  Copyright © 2018 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2018, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import UIKit
+import LocalAuthentication
+
+// MARK: - Security UserDefaults keys
+public let SecuritySettingsKey: String = "security-settings"
+public let SecuritySettingsFrequencyKey: String =  "security-settings-frequency"
+public let SecuritySettingsPasscodeKey: String = "security-settings-usePasscode"
+public let SecuritySettingsBiometricalKey: String = "security-settings-useBiometrical"
+
+// MARK: - Section identifier
+private let SecuritySectionIdentifier: String = "settings-security-section"
```

Comment:
Please consider just using it in place if an identifier is only used once and private to a class.